src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1256,6 +1269,7 @@ class ContentExplorer extends Component<Props, State> {
      * @return {void}
      */
     changeViewMode = (viewMode: ViewMode): void => {
+        this.store.setItem(localStoreViewMode, viewMode);
         this.setState({ viewMode });
```

Comment:
Ideally, we only want to track state in one place to avoid them getting out of sync. Can we eliminate `this.state.viewMode` entirely and just call `this.store.getItem` each time?